libs/components/src/lib/experience/index.tsx

```diff
@@ -1,27 +1,129 @@
 import { FC } from "react";
-
-import { TextField } from "../TextField";
+import cls from "classnames";
+// components
+import { TextArea, TextField } from "@bright-resume/components";
+// types
+import { ExperienceChildProps, ExperienceProps } from "../types/index.type";
+// icons
+import { AddCircleRoundedIcon, RemoveCircleRounded } from "../Icons";
 // locals
 import { useData } from "./useData";
 import classes from "./index.module.scss";
 
 /* eslint-disable-next-line */
-export interface ExperienceProps {}
 
-export const Experience: FC<ExperienceProps> = () => {
+export const Experience: FC<ExperienceProps> = ({
+  header = { placeholder: "Experience" },
+  items = [
+    {
+      id: "child-1",
+      position: { placeholder: "Position", label: "position" },
+      company: { placeholder: "company name", label: "company" },
+      description: {
+        label: "description",
+        placeholder:
+          "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+      },
+    },
+  ],
+  onIncrease = () => null,
+  onDecrease = () => null,
+}) => {
   const data = useData();
-  return (
-    <div className={classes.container}>
-      <div className={classes.wrapper}>
-        <h1 className={classes.heading1}>Experience</h1>
-        <div className={classes.text}>
-          <div className={classes.dot} />
+
+  const renderHeader = () => {
+    return (
+      <div className={classes.header__container}>
+        <TextField
```

Comment:
I think you forget pass headerArgs to TextField (...header)

Reply:
you are right, this issue has been fix in this pull request: https://github.com/ErfanSeidipoor/bright-resume/pull/87